pylxd/models/profile.py

```diff
@@ -17,10 +17,11 @@
 class Profile(model.Model):
     """A LXD profile."""
 
-    name = model.Attribute(readonly=True)
-    description = model.Attribute()
     config = model.Attribute()
+    description = model.Attribute()
     devices = model.Attribute()
+    name = model.Attribute(readonly=True)
+    used_by = model.Attribute(readonly=True)
```

Comment:
It would be a good idea to add `'used_by': [],` to `def profile_GET(...)` in the `pylxd/tests/mock_lxd.py` file to ensure the test cases match what's coming back.